site/gatsby-node.js

```diff
@@ -0,0 +1,21 @@
+const path = require(`path`)
+const { kebabCase } = require('lodash')
+
+exports.createPages = ({ graphql, boundActionCreators: { createPage } }) => {
+  // Add a page for each documented module, passing through the documentation data generated from
+  // source as a property of the page's pathContext.
+  //
+  // See: https://www.gatsbyjs.org/docs/creating-and-modifying-pages/
+
+  const json = require('./docs.json')
+
+  json.children.forEach((moduleDocJson) => {
+    createPage({
+      path: `/modules/${kebabCase(moduleDocJson.name)}`,
```

Comment:
This is the first time I've encountered the term 'kebabCase' - good stuff - probably even better after a few beers...

Reply:
-yes-it-is-
^^^^^^^^^^

Reply:
+1
The game has changed.